### ISSUE TITLE
Restbase URL could be more clearly shown

### ISSUE THREAD

**Issue by gee-one** [physikerwelt/mathoid-docs#1]
The documentation is the best I have found for setting up a small private wiki.  It would be a bit clearer to illustrate the final step.

$wgMathFullRestbaseURL = 'http://<your domain>:7231/localhost/'; # IP of Mathoid server

I had to find this through trial and error, using the API at http://<your domain>:7231/localhost/v1/ and the troubleshooting tips from https://www.mediawiki.org/wiki/Extension:Math/Restbase.  I am certainly not an expert, so there is no guarantee that this will break other functions, but it is working for me.  One of your comments on phabricator was also a clue that indicated that the Restbase URL was used by the client to retrieve the image.

Basically, I used the check command POST and then fed the x-resource-location back into the GET command to see why I was having so much trouble connecting to restbase.  I could see that the mathoid and restbase were generating and storing a PNG image, but I couldn't get it to display.  

At any rate, the documentation steps are spot on, and just a little more spoon feeding is needed at the end to connect the wiki to the restbase. Thank you!
